src/content/blam/tags/bitmap/readme.md

```diff
@@ -14,55 +14,43 @@ Bitmaps are used for visuals that need textures or sprites like environments, ob
 
 [[toc]]
 
-## Basics
-To store textures and images in maps we use bitmap tags. Bitmap tags on their simplest are compiled from a .tif file in your data directory.
+# Basics
+To store textures and images in maps we use bitmap tags. Bitmap tags on their simplest are compiled from a `.tif` file in your data directory.
 
-diffuse map, bump map, multipurpose etc (i know detail map is like scratches overlay
-
-multipurpose handles detail masking, color change, self illumination and specularity
-detail is the red channel
-color change is the alpha channel, its a mask for color values specified in tags that get multiplied over the diffuse. color change source in shader model controls what values get used in things like biped or actor variant.
-self illum is green,
-specularity is blue, it masks cubemap reflections
-
-xbox channel sources are different. red is specular, blue is color change, and alpha is detail.
-
-you dont need to have a multi for detail maps to show
-
-### Commands
+### Tool Commands
```

Comment:
You have no h2 in this document -- h1 and h2 will show up in the ToC so I suggest changing these h3 to h2 where appropriate for easier navigation.

Reply:
Looks like an oversight that there's only h1 and h3. I'd be fine changing it in either way

Reply:
The formats section would benefit from it especially